### ISSUE TITLE
Comment from Ceapat (Spain) Use of numbers. 4.4.13

### ISSUE THREAD

**Issue by claraceapat** [w3c/coga#238]
Dear collegues,

Safety and accessibility

Regarding the use of numbers, it should avoid requesting that the person have to make a math operation to access a service.

To offer safety to the customer in their online payments, before making the purchase, sometimes mathematical operations of the following type are requested;

To finalize your purchase, enter the result of multiplying positions 1 and 2 of your PIN and add 5400 to the result.

This type of statement will guarantee the safety of the transaction, but it is difficult to understand by people with congitive disabilities.

Both safety and cognitive accessibility should be guaranteed.

Thanks

**Comment by rachaelbradley** [comment]
Thank you for your suggestion. We have included it in 4.7.5 and added your example!

**Comment by claraceapat** [comment]
Hi from Spain!
Thanks for considering it.
Lately my bank is the way it uses to secure online payments, so the example is real and true!
Thanks you so much!
